### DISCUSSION TITLE
Lighting?

### DISCUSSION THREAD

**Discussion by DavidIgor** [cedricp/ddt4all#996] [General]
what is this option? 
![image](https://github.com/cedricp/ddt4all/assets/122531259/5335badc-9ee5-410b-9ae8-0fb2a7a79818)

**Comment by KarelSvo** [comment]
Light control via energy management module - false 
Your lighting is controlled without a control device. Just switches, cables, fuses and bulbs.

**Reply by DavidIgor** [reply]
@KarelSvo this is only option under mmc when I hit scan ... if I choose my car and module manually there is more options, but will they work-why scan won't see other options in this module? 
same applies for rest ... for example throttle body

**Reply by DavidIgor** [reply]
@KarelSvo should I set it to "True"? then maybe welcome lights or cornernering lights (using fog lights) could be set????
